### DISCUSSION TITLE
Misleading text ("Open Source") on the pricing page

### DISCUSSION THREAD

**Discussion by l2dy** [outline/outline#2819] [General] [answered]
The self-managed option on https://www.getoutline.com/pricing claims to have an Open Source plan, but the BSL 1.1 license explicitly says that it is not an Open Source license. Could you fix the wording on getoutline.com to be more clear about this?

<img width="806" alt="open-source" src="https://user-images.githubusercontent.com/14329097/144709376-0f08b211-bb47-4e63-b3ab-72a995265aac.png">

> The Business Source License (this document, or the “License”) is not an Open
> Source license. [...]

https://github.com/outline/outline/blob/a674a8668bf1ba479d0c9107bf750a3eddc02e01/LICENSE#L27-L29

**Comment by tommoor** [comment, marked as answer]
Changed to "Community Edition" ✅
